dev/generate.js

```diff
@@ -0,0 +1,16 @@
+const { createEnv } = require('yeoman-environment');
+
+const AVAILABLE_GENERATORS = ['new-rule', 'rules-index'];
+
+async function main() {
+  let generator = process.argv.slice(2);
+  let env = createEnv();
+
+  for (const generator of AVAILABLE_GENERATORS) {
+    env.register(require.resolve(`./${generator}-generator`), `ember-template-lint:${generator}`);
+  }
```

Comment:
It's kinda odd to use `const generator` here to shadow the `let generator` just above, maybe change one of them?

Reply:
Ya good call.

Reply:
Fixed.